cmd/crictl/portforward.go

```diff
@@ -36,6 +37,14 @@ var runtimePortForwardCommand = &cli.Command{
 	Name:      "port-forward",
 	Usage:     "Forward local port to a pod",
 	ArgsUsage: "POD-ID [LOCAL_PORT:]REMOTE_PORT",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    transportFlag,
+			Aliases: []string{"r"},
+			Value:   transportSpdy,
+			Usage:   fmt.Sprintf("Transport protocol to be used, must be one of: %s, %s", transportSpdy, transportWebsocket),
```

Comment:
To align with the other help formats.

```suggestion
			Usage:   fmt.Sprintf("Transport protocol to use, can be one of: %s|%s", transportSpdy, transportWebsocket),
```

Even closer to the format:

```suggestion
			Usage:   fmt.Sprintf("Transport protocol to use, one of: %s|%s", transportSpdy, transportWebsocket),
```


Also, moved the "must" to "can" as there is a default set, which makes this option optional.

Reply:
Fixed as suggested :+1: